tao_pegtl_vendor/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>tao_pegtl_vendor</name>
+  <version>0.0.1</version>
+  <description>
+    Wrapper around pegtl, providing nothing but a dependency on pegtl, on some systems.
+    On others, it provides an ExternalProject build of spdlog.
```

Comment:
Why this package is related to `spdlog`?

Reply:
Sorry, I forgot to update it. Updated in https://github.com/ros2/rcl_content_filter_fallback/pull/1/commits/850727f24abff68c42bdf6ca17b2da434f5199bc